work/powex.jsonld

```diff
@@ -5,38 +5,32 @@
 
     "id": "@id",
     "type": "@type",
-        
+
     "ProofOfWorkExperience": {
       "@id": "https://github.com/sunbird-specs/vc-specs/work#ProofOfWorkExperience",
       "@context": {
         "version": "0.1",
         "@protected": true,
-        
+
         "id": "@id",
         "type": "@type",
-        
+
         "cred": "https://www.w3.org/2018/credentials#",
         "dc": "http://purl.org/dc/terms",
+        "sunvc": "https://github.com/sunbird-specs/vc-specs#"
         "powex": "https://github.com/sunbird-specs/vc-specs/work#",
         "schema": "https://www.schema.org",
         "sec": "https://w3id.org/security#",
         "xsd": "http://www.w3.org/2001/XMLSchema#",
-       
-        "certificateId": "@id",
+
         "issuer": { "@id": "cred:issuer", "@type": "schema:Organisation" },
         "legalName": "schema:legalName",
-        "address": { "@id": "schema:address", "@type": "schema:PostalAddress" },
-        "issuedOn": "cred:issuanceDate",
-        "credentialSubject": { "@id": "cred:credentialSubject", "@type": "schema:Person" },
-        "name": "schema:name",
-        "jobTitle": "schema:jobTitle",
+        "address": { "@id": "schema:address", "@type": "sunvc:Address" },
```

Comment:
Some SBVC core objects like 'fullname' starts with small letter while objects like 'Address' starts with capital. Any reason? If it needs fix can you please fix it aa well @kochhar ? Thanks. @dileepbapat